README.md

```diff
@@ -69,6 +69,7 @@ Communication with the backend from QML to C++ and vice-versa is always and excl
 - Now that the FSM receives the `backend` by reference, should the FSM be directly responsible of carrying out back-end actions such as reinitialising the game board? Or is it better that the FSM just informs the `Backend` that the `Backend` has to perform a certain action, and the `Backend` actually carries out that action?
 - The automatic stop is probably inefficient, because it checks for equality between the `m_currentMatrix` and the `m_nextMatrix` at every game loop. Should a timer be used to rarify the checks?
 - Add the option to enable / disable the cell animation, and to set the percentage with respect to the game timer period.
+- Consider if GameStateMachine and other classes should be singletons
```

Comment:
Consider adding already discussed TODO: using an asymmetrical fading effect (i.e. only a fade-in, not a fade-out).

Reply:
Added discussed TODO item and amended commit.

Reply:
OK